src/rules/orderedImportsRule.ts

```diff
@@ -116,6 +116,7 @@ export class Rule extends Lint.Rules.AbstractRule {
         "Import sources of different groups must be sorted by: libraries, parent directories, current directory.";
     public static IMPORT_SOURCES_UNORDERED = "Import sources within a group must be alphabetized.";
     public static NAMED_IMPORTS_UNORDERED = "Named imports must be alphabetized.";
+    public static IMPORT_SOURCES_OF_SAME_TYPE_NOT_IN_ONE_GROUP = "Import sources of the same type must be grouped together.";
```

Comment:
the word `type` here confused the heck out of me! you want to group my `string` imports separately from `functions`??

then i figured out what you mean. is there a word other than `type` we can use to describe this? perhaps `source`?

Reply:
build will pass after you push a fix for this language :)

Reply:
@giladgray Thank you for the review. I agree the word `type` is to general, however "Import sources of the same source..." sounds even weirder imo :/ I took the term `Import source` from the existing error message two lines up.

I decided to add an explanation to the error message. Do you think it is clearer now?